packages/dbgpt-app/src/dbgpt_app/openapi/api_v1/tools/execute_tool.py

```
@@ -15,6 +18,58 @@ async def execute_tool(tool_name: str, args: dict) -> str:
+                registry = connector_manager.get_confirmation_registry()
+                if interceptor.should_confirm(tool_name, args):
+                    import asyncio
+                    import uuid
+
+                    confirm_id = str(uuid.uuid4())
+                    registry.register(confirm_id)
+                    _PENDING_CONFIRMATIONS[confirm_id] = {
+                        "confirm_id": confirm_id,
+                        "tool_name": tool_name,
+                        "args_summary": interceptor._summarize_args(args),
+                        "message": f"即将执行写操作 {tool_name}，是否确认？",
```

Ruff (0.15.21)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)